sky/spot/spot_utils.py

```diff
@@ -231,14 +231,17 @@ def stream_logs_by_id(job_id: int, follow: bool = True) -> str:
     backend = backends.CloudVmRayBackend()
     spot_status = spot_state.get_status(job_id)
```

Comment:
The signature of this func suggests it can return None. Do we need to guard against this case, `spot_status == None`?

Reply:
Good catch! Added. Thanks!

Reply:
Good catch! Added. Thanks!

Reply:
Looks good, thanks. Can we document something like "spot_status can be None when ..."?